### ISSUE TITLE
year and month partly obfuscated using Chrome when using selectMonths or selectYears options

### ISSUE THREAD

**Issue by rounders** [amsul/pickadate.js#1213]
While using chrome 83.0.4103.116 on a mac (haven't tried on windows)
visit the date examples page: 
https://amsul.ca/pickadate.js/date/

scroll down to the 'accessibility labels' example and try the example. As you can see in the
screenshot below the the year and month options are partly obfuscated. 

<img width="881" alt="Screen Shot 2020-07-10 at 6 36 58 PM" src="https://user-images.githubusercontent.com/71225/87213789-5bfb2200-c2dc-11ea-958a-3e554fec5b1e.png">

**Comment by betheymc** [comment]
Yes I noticed this on Linux on Chrome as well. It seems like wherever the dropdown is being used on months or years, you can't see part of the month and year text.

**Comment by dave-stan** [comment]
Any news on this problem yet? It just doesn't look professional when using Chrome.

**Comment by alfredog1976** [comment]
You can set the height to the following in "default.date.css":
.picker__select--month, .picker__select--year {
	height: 2.35em;
}

I've tested this on Firefox/Chrome/Safari on a Mac as well as Firefox/Chrome/Edge on Windows. Don't see any issues after the change.
